scipy/stats/tests/test_stats.py

```diff
@@ -4515,46 +4514,40 @@ def test_friedmanchisquare():
                  0.866, 1.0, 0.965, 0.614, 0.9751, 0.946]),
           array([0.798, 0.569, 0.967, 0.657, 0.898, 0.931, 0.685, 0.625,
                  0.875, 1.0, 0.962, 0.669, 0.975, 0.970])]
+    ref1 = (10.2283464566929, 0.0167215803284414)
 
     # From "Bioestadistica para las ciencias de la salud" Xf=18.95 p<0.001:
-    x2 = [array([4,3,5,3,5,3,2,5,4,4,4,3]),
-          array([2,2,1,2,3,1,2,3,2,1,1,3]),
-          array([2,4,3,3,4,3,3,4,4,1,2,1]),
-          array([3,5,4,3,4,4,3,3,3,4,4,4])]
+    x2 = [array([4, 3, 5, 3, 5, 3, 2, 5, 4, 4, 4, 3]),
+          array([2, 2, 1, 2, 3, 1, 2, 3, 2, 1, 1, 3]),
+          array([2, 4, 3, 3, 4, 3, 3, 4, 4, 1, 2, 1]),
+          array([3, 5, 4, 3, 4, 4, 3, 3, 3, 4, 4, 4])]
```

Comment:
Nit, can we just make this a list of lists since the first thing that happens to the arrays is to call `.tolist()` on them